java-frontend/src/main/java/org/sonar/java/se/SymbolicExecutionVisitor.java

```diff
@@ -64,7 +68,7 @@ public void visitNode(Tree tree) {
   }
 
   public void execute(MethodTree methodTree) {
-    ExplodedGraphWalker walker = egwFactory.createWalker(behaviorCache, (SemanticModel) context.getSemanticModel());
+    ExplodedGraphWalker walker = egwFactory.createWalker(behaviorCache, semanticModel);
```

Comment:
This makes me wonder a bit if we are not creating a new way to access semantic model rather than relying on what's existing.

Reply:
I reworked it, so semantic model is acquired from scanner context